packages/react-ui-kit/src/Form/Tooltip.tsx

```diff
@@ -1,6 +1,6 @@
 /*
  * Wire
- * Copyright (C) 2022 Wire Swiss GmbH
+ * Copyright (C) 2024 Wire Swiss GmbH
```

Comment:
this should remain 2022
```suggestion
 * Copyright (C) 2022 Wire Swiss GmbH
```